pkg/api/entries.go

```diff
@@ -301,9 +306,64 @@ func createLogEntry(params entries.CreateLogEntryParams) (models.LogEntry, middl
 	logEntry := models.LogEntry{
 		entryID: logEntryAnon,
 	}
+
+	if api.newEntryPublisher != nil {
+		// Publishing notifications should not block the API response.
```

Comment:
We should think through how we'll detect errors. We've missed failed Redis requests before. We'll also need a script to handle backfills.

FWIW, even if this were blocking, we'd still need to handle backfills if the request to Trillian succeeds but the topic write fails.

@bobcallaway Thoughts on blocking vs non-blocking?

Reply:
I'm good with marking this as resolved, as the GCP Datastream approach will solve this.